lib/ansible/modules/windows/win_get_url.py

```diff
@@ -73,6 +73,40 @@
     type: bool
     default: yes
     version_added: '2.4'
+  checksum:
+    description:
+      - 'If a checksum is passed to this parameter, the digest of the
+        destination file will be calculated after it is downloaded to ensure
+        its integrity and verify that the transfer completed successfully.
+        Format: <checksum>, e.g. checksum="D98291AC[...]B6DC7B97"'
+      - Additionally, if a checksum is passed to this parameter, and the file exist under
+        the C(dest) location, the I(destination_checksum) would be calculated, and if
+        checksum equals I(destination_checksum), the file download would be skipped
+        (unless C(force) is C(true)).
```

Comment:
Please use yes/no values, as that's what the documentation is using as well.
```suggestion
        (unless C(force) is C(yes)).
```

Reply:
Fixed